harmonica/_transformations.py

```diff
@@ -393,6 +393,69 @@ def total_gradient_amplitude(grid):
     return np.sqrt(gradient[0] ** 2 + gradient[1] ** 2 + gradient[2] ** 2)
 
 
+def tilt(grid):
```

Comment:
@leomiquelutti is this usually just called "tilt" in the literature? I don't read this type of paper much so I'm not sure. But I've seen it as "tilt angle" a few times.

Reply:
I don't know but naming it _tilt angle_ instead of _tilt_ looks better. Can I change it @leouieda?

Reply:
Yep, please feel free to change it!